index.html

```diff
@@ -192,10 +217,7 @@ <h2 class="contenthead">Professional Art Printing Data</h2>
 <h2 class="aboutmehead">About me</h2>
 <p class="aboutpara">Hello I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
 <br>
-<button type="button" class="buttn">Get My Resume</button>
-
-<article class="container2">
-    <div class="aboutinfo">
+        <button type="button" class="buttn">Get My Resume</button>
 </div>
 </div>
 <hr class="line">
```

Comment:
And in here it's recommended to follow the same font.

![image](https://user-images.githubusercontent.com/34779928/194695795-ef6671bd-313b-4b7b-ae29-a98917dbfbe6.png)